### PR TITLE
add debug to node sidebar

### PR DESCRIPTION
**Pull request by ambuznego**

### CODE REVIEW

File: assets/js/components/common/Debug.jsx [hunk 1]
```diff
@@ -114,28 +165,31 @@ class Debug extends Component {
...
-              </span>
-            ))
-          }
+          {data.map((d) => (
+            <span key={d.id}>
+              <DebugEntry
+                event={d}
+                clearSingleEntry={this.clearSingleEntry}
+                expandAll={expandAll}
+                width={this.props.entryWidth || 600}
```
Comment (ambuznego) [inline assets/js/components/common/Debug.jsx:179]: main change in this file to allow for a smaller debug entry when opening from node workspace

File: assets/js/components/common/DebugEntry.jsx [hunk 2]
```diff
@@ -110,260 +127,329 @@ class DebugEntry extends Component {
...
+      <div
+        key={event.id}
+        style={{
+          paddingLeft: 20,
+          paddingRight: 20,
+          paddingTop: 10,
+          paddingBottom: 10,
+          marginBottom: 10,
+          marginLeft: 25,
+          width: this.props.width,
```
Comment (ambuznego) [inline assets/js/components/common/DebugEntry.jsx:157]: main change here

File: assets/js/components/common/Debug.jsx (hunk 1 above)
Comment (vicmgs) [reply assets/js/components/common/Debug.jsx:179]: ty

**Review by vicmgs — APPROVED** [review]
nice

**Pull request merged**